### ISSUE TITLE
Redesigned homepage: Add alt property to infinity symbol

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3793]
### Overview
On the redesigned homepage, we want to add an `alt` property to the infinity symbol image under the Technology section so that the image is Web Content Accessibility Guidelines (WCAG) compliant.

### Details
<details>
  <summary>Click here to see a screenshot of the redesigned homepage's Technology section</summary>
  <img width="821" alt="" src="https://user-images.githubusercontent.com/31293603/210594926-7c677b90-dc0e-4f11-bee0-d705858ad312.png">
</details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch`. Please follow ["Step 1: Create your branch" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) wiki article.
- [x] In VS Code or another IDE on your computer, navigate to `_includes/home-page/home-technologies.html` and change the following from
  ```
  <img src="assets/images/hero/infinite.svg">
  ```
  to
  ```
  <img src="assets/images/hero/infinite.svg" alt="infinite">
  ```
- [x] Using developer tools in your browser, on the redesigned homepage,
  - [x] Check the `alt="infinite"` property shows up for the infinity symbol image under the Technology section.
  - [x] Check your changes do not break anything on the website in mobile, tablet, and desktop views
- [x] To update your topic branch, please follow ["Step 2: Update your branch with upstream changes" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-2-update-your-branch-with-upstream-changes) wiki article.
- [ ] The pull request created from this issue should be merged into `feature-homepage-launch`. Please follow ["Step 3: Create a pull request" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-3-create-a-pull-request) wiki article

**Tech Lead or Merge Team**
- [ ] After the pull request associated with this issue is approved and merged, please make sure this issue is closed. If it is not closed by the automation, leave a new comment stating what pr number fixes this issue and then close this issue as completed.

### Resources/Instructions
- [How to work off of a feature branch wiki page](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [_includes/home-page/home-technologies.html on feature-homepage-launch branch](https://github.com/hackforla/website/blob/feature-homepage-launch/_includes/home-page/home-technologies.html)

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty On the redesigned homepage's Technology section's infinity symbol image, there is currently no alt property for it. Did you want it to be `alt="infinite"` or some other alt text description? Ans: Yes

<details>
  <summary>Click here to see a screenshot of the redesigned homepage's Technology section</summary>
  <img width="821" alt="" src="https://user-images.githubusercontent.com/31293603/210594926-7c677b90-dc0e-4f11-bee0-d705858ad312.png">
</details>

**Comment by jch1013** [comment]
Availability - flexible
ETA - 1/24 end of day

**Comment by JessicaLucindaCheng** [comment]
- PR #3865 fixed the issue.
